testing/trino-product-tests/src/main/java/io/trino/tests/product/phoenix/TestPhoenix.java

```diff
@@ -28,8 +28,7 @@
 public class TestPhoenix
         extends ProductTest
 {
-    // TODO: https://github.com/trinodb/trino/issues/21824
-    @Test(groups = {PHOENIX, PROFILE_SPECIFIC_TESTS}, timeOut = 30_000L, enabled = false)
+    @Test(groups = {PHOENIX, PROFILE_SPECIFIC_TESTS})
```

Comment:
> Run Phoenix product test after 5.2.1 upgrade

You can change the commit title to "Enable Phoenix product test".

Reply:
@virajjasani You renamed the PR title only, please also update the commit message. Looks good to merge.

Reply:
Done. Thank you @ebyhr @hashhar!!